zaza/openstack/charm_tests/test_utils.py

```diff
@@ -133,6 +133,29 @@ def setUpClass(cls, application_name=None, model_alias=None):
             model_name=cls.model_name)
         logging.debug('Leader unit is {}'.format(cls.lead_unit))
 
+    def config_current(self, application_name=None, keys=None):
+        """Get Current Config of an application normalized into key-values.
+
+        :param application_name: String application name for use when called
+                                 by a charm under test other than the object's
+                                 application.
+        :type application_name: str
+        :param keys: iterable of strs to index into the current config.  If
+                     None, return all keys from the config
+        :type keys: Union[iterable[str], None]
```

Comment:
It's great that you are using types, and an easier/simpler type here is: `Optional[iterable[str]]` which is the same as `Union[iterable[str], None]`

This is also missing the :return: and :rtype: items in the docstring for the return type.